plugins/woocommerce-admin/client/task-lists/fills/PaymentGatewaySuggestions/index.js

```diff
@@ -245,7 +249,8 @@ export const PaymentGatewaySuggestions = ( { onComplete, query } ) => {
 		<List
 			heading={ additionalSectionHeading }
 			headingDescription={ additionalSectionHeadingDescription }
-			recommendation={ recommendation }
+			// No recommendation if WooPayments is supported (and displayed).
+			recommendation={ isWCPaySupported ? false : recommendation }
```

Comment:
The original requirement was about removing the recommended badge from just Stripe, but this code change will remove the badge from all payment providers (including PayPal). Is this intentional? cc @timmy5685

Reply:
Yes, it is intentional as the intent is not to confuse users with multiple recommendations (WooPayments and something else). Stripe is just the one that usually got that badge because it was the highest priority. I've confirmed with Tim that the flows are as intended.